pinot-query-planner/src/main/java/org/apache/pinot/query/planner/StageMetadata.java

```diff
@@ -38,16 +39,13 @@
  *   <li>the server instances to which this stage should be execute on</li>
  * </ul>
  */
-public class StageMetadata implements Serializable {
+public class StageMetadata {
   private List<String> _scannedTables;
 
   // used for assigning server/worker nodes.
-  private List<VirtualServer> _serverInstances;
+  private Map<ServerInstance, List<Integer>> _serverToWorkerIdsMap;
```

Comment:
Do we need this map? Ideally we should already split the segments to the worker

Reply:
Tis map is used only during dispatch. And it will be no longer used in QueryRunner